.github/renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "addLabels": ["ok-to-test"],
+    "rebaseWhen": "behind-base-branch",
```

Comment:
Clarifying: Is this to confirm that the CI always passes even if a PR is merged in the interim?

Reply:
This field will allow the CI to`rebase` the PR to include the latest changes if the base branch becomes out of sync:

> Renovate will rebase whenever the branch falls 1 or more commit behind its base branch.

Reply:
I've set this to `auto` which is the recommended option.